grammars/jsdoc.cson

```diff
@@ -0,0 +1,478 @@
+        |method|module|mixes|mixin|name|requires|see|this|typedef|uses)
+      )
+      \\s+
+      (
+        (?:
+          [^{}@\\s*] | \\*[^/]
+        )+
+      )
+    '''
+  }
+  {
+    # Tags followed by a quoted arbitrary value
+    # -  @<tag> "Quoted value"
+    'begin': '((@)(?:default(?:value)?|license|version))\\s+([\'"])'
+    'beginCaptures':
+      '1':
+        'name': 'storage.type.class.jsdoc'
+      '2':
+        'name': 'punctuation.definition.block.tag.jsdoc'
+      '3':
+        'name': 'variable.other.jsdoc'
+    'contentName': 'variable.other.jsdoc'
+    'end': '\\3|(?=$|\\*/)'
+    'endCaptures':
+      '0':
+        'name': 'variable.other.jsdoc'
+  }
+  {
+    # Tags followed by an arbitrary value
+    # -  @<tag> value
+    'captures':
+      '1':
+        'name': 'storage.type.class.jsdoc'
+      '2':
+        'name': 'punctuation.definition.block.tag.jsdoc'
+      '3':
+        'name': 'variable.other.jsdoc'
+    'match': '((@)(?:default(?:value)?|license|tutorial|variation|version))\\s+([^\\s*]+)'
+  }
+  {
+    # Tags without arguments, or a tag without expected arguments. Because JSDoc permits
+    # tags to be spread across lines, we should at least highlight the opening tag for
+    # stuff like this:
+    #
+    #    /**
+    #      * @param
+    #      * {type}
+    #      * name
+    'captures':
+      '1':
+        'name': 'punctuation.definition.block.tag.jsdoc'
+    'match': '''(?x) (@)
+      (?:abstract|access|alias|arg|argument|async|attribute|augments|author|beta|borrows|bubbles
+      |callback|chainable|class|classdesc|code|config|const|constant|constructor|constructs|copyright
+      |default|defaultvalue|define|deprecated|desc|description|dict|emits|enum|event|example|exception
+      |exports?|extends|extension(?:_?for)?|external|externs|file|fileoverview|final|fires|for|func
+      |function|global|host|ignore|implements|implicitCast|inherit[Dd]oc|inner|instance|interface|kind
+      |lends|license|listens|main|member|memberof!?|method|mixes|mixins?|modifies|module|name|namespace
+      |noalias|nocollapse|nocompile|nosideeffects|override|overview|package|param|preserve|private|prop
+      |property|protected|public|read[Oo]nly|record|require[ds]|returns?|see|since|static|struct|submodule
+      |summary|suppress|template|this|throws|todo|tutorial|type|typedef|unrestricted|uses|var|variation
+      |version|virtual|writeOnce)
+      \\b
+    '''
+    'name': 'storage.type.class.jsdoc'
+  }
+  {
+    # Description preceding {@inline tag}
+    'captures':
+      '1':
+        'name': 'punctuation.definition.bracket.square.begin.jsdoc'
+      '2':
+        'name': 'punctuation.definition.bracket.square.end.jsdoc'
+    'match': '(\\[)[^\\]]+(\\])(?={@(?:link|linkcode|linkplain|tutorial))'
+    'name': 'constant.other.description.jsdoc'
+  }
+  {
+    # {@link|tutorial …}
+    'begin': '({@)(?:link|linkcode|linkplain|tutorial)\\s*'
```

Comment:
The bracket needs to be separated from the at-sign.  Also, the comment is a bit different from the regex match here.  It looks like the comment is specifying `{@link}` or `{tutorial}`, but the match is specifying `{@link}`, `{@tutorial}`.